ida/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM intezer/ida-base
+
+ARG IDA_INSTALLATION_FILE=ida.run
+ARG IDA_PASSWORD
+
+COPY $IDA_INSTALLATION_FILE /installation/
+COPY requirements.txt /installation/
+
+# Installing requirements for image
+RUN pip2 install -r /installation/requirements.txt
+
+# Install IDA
+RUN mkdir /ida
+# Copy IDAPython helper script to IDA installation folder
+COPY idal.py /ida/
```

Comment:
I saw in the spotify cassandra they are copying their scripts to `/usr/local/bin/`instead of creating a directory then adding the new dir to PATH env variable